questions/html-questions.md

```diff
@@ -11,5 +11,5 @@
 * What is progressive rendering?
 * Why you would use a `srcset` attribute in an image tag? Explain the process the browser uses when evaluating the content of this attribute.
 * Have you used different HTML templating languages before?
-* What's your process for addressing browser-specific rendering problems? Do you find that a certain browser is more challenging to work with than others?
-* Explain Why Table-less Layout Is Very Important?
+* What are ‘web workers’?
+* How do I hide my source?
```

Comment:
Hmm, what do you mean?

Reply:
I mean by 2nd ques:- Is there a way we can hide my source code, so that it cannot be copied.

Reply:
obviously - no. Let's wait for others opinion ))